app/src/main/java/com/naman14/timberx/util/NotificationUtils.kt

```diff
@@ -30,13 +30,13 @@ import androidx.palette.graphics.Palette
 import com.naman14.timberx.R
 import com.naman14.timberx.TimberMusicService
 import com.naman14.timberx.ui.activities.MainActivity
-import com.naman14.timberx.util.media.isPlaying
+import com.naman14.timberx.util.extensions.isPlaying
 
 object NotificationUtils {
 
     private const val CHANNEL_ID = "timberx_channel_01"
     private var mNotificationPostTime: Long = 0
```

Comment:
Why `m` here but not elsewhere? :stuck_out_tongue:

Reply:
@Zhuinden pre-existing code - I personally hate Hungarian notation

Reply:
You removed it in the adapter :thinking:

Reply:
Yeah I guess I could do it here as part of the "cleanup"

Reply:
But there's Hungarian all over the place so this PR could definitely get too big